src/core_fs/event_bus/mod.rs

```diff
@@ -0,0 +1,5 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
```

Comment:
If the `mod.rs` is on there for exposing the other file, why do we need a directory instead of just an `event_bus.rs` file ?

Reply:
Because it prepares for other event_bus

Reply:
Done